spec/requests/api/querying_spec.rb

```diff
@@ -599,7 +598,7 @@ def create_vms_by_name(names)
       run_get vms_url, :expand => "resources,software"
 
       expect_query_result(:vms, 1, 1)
-      expect_result_resources_to_include_keys("resources", %w(id href guid name vendor software))
```

Comment:
Ah, I see they were added back in the next commit. Is there a way of removing these changes from the history? Is this commit only :green_apple: with this change?